include/eos/morphablemodel/MorphableModel.hpp

```diff
@@ -353,7 +356,8 @@ class MorphableModel
         if (has_texture_coordinates())
         {
             mesh = sample_to_mesh(shape_sample, color_sample, shape_model.get_triangle_list(),
-                                  color_model.get_triangle_list(), texture_coordinates);
+                                  color_model.get_triangle_list(), texture_coordinates,
+                                  texture_triangle_indices);
```

Comment:
it feels a bit inconsistent, that there is triangle list everywhere but here precomputed indices are used

Reply:
This is commit from origin repo